lcov.base.info

```diff
@@ -0,0 +1,202 @@
+SF:lib/bloc/authentication/signup/google_signup_state.dart
+DA:10,0
+DA:16,1
+DA:23,2
+DA:24,0
+DA:25,0
+DA:26,0
+DA:28,0
+LF:7
```

Comment:
Please do not include this file in the commit. Maybe I’ll need to update gitignore to ignore this file